var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -17,7 +17,13 @@ class Pfunit(CMakePackage):
 
     maintainers = ['citibeth']
 
-    version('3.2.9', 'e13d8362284b13b7c863e2fe769a9d5c')
+    # version('4.0.0',  sha256='b8b6470f2b1e2b19c164c244c10e803bd69c8da9a6a5a65ba7c479fb8b92a1e1')
```

Comment:
Please add brief comment explaining why it is commented out, and linking to the discussion on it.